src/Orleans.Persistence.Cassandra/Orleans.Persistence.Cassandra.csproj

```diff
@@ -3,8 +3,8 @@
     <TargetFramework>netstandard2.0</TargetFramework>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="CassandraCSharpDriver" Version="3.6.0" />
-    <PackageReference Include="Microsoft.Orleans.Core" Version="$(OrleansPackageVersion)" />
-    <PackageReference Include="Microsoft.Orleans.Runtime.Abstractions" Version="$(OrleansPackageVersion)" />
+    <PackageReference Include="CassandraCSharpDriver" Version="3.9.0" />
+    <PackageReference Include="Microsoft.Orleans.Core" Version="2.3.0" />
+    <PackageReference Include="Microsoft.Orleans.Runtime.Abstractions" Version="2.3.0" />
```

Comment:
This value should be changed in [Directory.Build.props](https://github.com/OrleansContrib/Orleans.Persistence.Cassandra/blob/master/Directory.Build.props) instead

Reply:
Agreed